.github/workflows/e2e.container.push.branch1.default.slsa3.yml

```
@@ -50,7 +50,7 @@ jobs:
 
           BRANCH=$(e2e_this_branch)
           if [[ "$BRANCH" == "$GITHUB_REF_NAME" ]]; then
-            echo "::set-output name=continue::yes"
+            echo "continue=yes" >> $GITHUB_OUTPUT
```

nit: quote the $GITHUB_OUTPUT. I don't think it would have spaces in it but just to be sure. Same for all others.
```suggestion
            echo "continue=yes" >> "${GITHUB_OUTPUT}"
```